ferrischat_webserver/src/guilds/edit_guild.rs

```diff
@@ -13,32 +13,44 @@ pub async fn edit_guild(
     let bigint_guild_id = u128_to_bigdecimal!(guild_id);
 
     let GuildUpdateJson { name } = guild_info.0;
-
-    let db = get_db_or_fail!();
-
-    let resp = sqlx::query!(
-        "UPDATE guilds SET name = $1 WHERE id = $2 RETURNING *",
-        name,
+    
+    sqlx::query!(
+        "SELECT * FROM guilds WHERE id = $1",
         bigint_guild_id
     )
     .fetch_optional(db)
     .await;
+    
+    match name {
+        Some(name) => {
+            let db = get_db_or_fail!();
+
+            let resp = sqlx::query!(
+                "UPDATE guilds SET name = $1 WHERE id = $2 RETURNING *",
+                name,
+                bigint_guild_id
+            )
+            .fetch_optional(db)
+            .await;
 
-    match resp {
-        Ok(resp) => match resp {
-            Some(guild) => HttpResponse::Ok().json(Guild {
-                id: bigdecimal_to_u128!(guild.id),
-                owner_id: bigdecimal_to_u128!(guild.owner_id),
-                name: guild.name.clone(),
-                channels: None,
-                members: None,
-            }),
-            None => HttpResponse::NotFound().json(NotFoundJson {
-                message: "Guild not found".to_string(),
-            }),
+            match resp {
+                Ok(resp) => match resp {
+                    Some(guild) => HttpResponse::Ok().json(Guild {
+                        id: bigdecimal_to_u128!(guild.id),
+                        owner_id: bigdecimal_to_u128!(guild.owner_id),
+                        name: guild.name.clone(),
+                        channels: None,
+                        members: None,
```

Comment:
Do another query to fetch the updated guild object after you've done the query to update it.